tests/notebooks/utils/gimvi_tutorial.py

```diff
@@ -0,0 +1,104 @@
+import umap
```

Comment:
Can we move these the source directory (i.e., `scvi`). Or what's our thinking on what goes where?

Reply:
In the repo, the notebooks are used both as tests as tutorials/demonstrations. For sake of readability, I refactored all the `utils` function of the `gimvi_tutorial` notebook in this file, rather than having huge cells of code. 
I thought it would be easier for the user to have demo notebooks and their supplementary materials in the same folder. What do you think?

Reply:
There are already some functions in the scvi source code folder that seem like they could just as easily go into this new `utils` folder. I'm concerned about having more than one place where helper functions could go. Not sure what the solution is.

Reply:
I agree with not having huge cells of code though!

Reply:
Is this utils code only useful for this one notebook? Or could it be useful for other notebooks too?

Reply:
Yes these are really specific pieces of codes for this one notebook with seq/spatial questions. I don't think they should go into the scvi source folder until we found some other usages in the future.